File: Challenge - Bromum 2018/answer.py
# you can write to stdout for debugging purposes, e.g.
# print("this is a debug message")

def solution(N, Q, B, C):
    # write your code in Python 3.6
    ball_colors = []
    for i in range(len(B)):
        ball_colors[B[i]][C[i]] += 1
        if(ball_colors[B[i]][C[i]] >= Q):
            return i

    return -1

if __name__== "__main__":
    print(solution(3, 2,  [1, 2, 0, 1, 1, 0, 0, 1], [0, 3, 0, 2, 0, 3, 0, 0]))
    print(solution(2, 2,  [0, 1], [5, 5]))
    print(solution(2, 2,  [0, 1, 0, 1, 0, 1], [1, 3, 0, 0, 3, 3]))

File: 6.Sorting - Triangle/answer.js
// you can write to stdout for debugging purposes, e.g.
// console.log('this is a debug message');

function solution(A) {
    // write your code in JavaScript (Node.js 8.9.4)
    A.sort((a,b) => a - b)
    for(let i = 0; i < A.length-2; i++){
        if(A[i] + A[i+1] > A[i+2]){
            if(A[i+1] + A[i+2] > A[i] && A[i] + A[i+2] > A[i+1]){
                return 1
            
            }
        }
    }
    
    return 0
}
